crates/jstz_cli/src/config.rs

```diff
@@ -189,25 +222,25 @@ impl Config {
     }
 
     /// Save the configuration to the file
-    pub fn save(&self) -> std::io::Result<()> {
+    pub fn save(&self) -> Result<()> {
         let path = Self::path();
-        if let Some(parent) = path.parent() {
-            fs::create_dir_all(parent)?;
+
+        if !path.exists() {
+            if let Some(parent) = path.parent() {
```

Comment:
what is the purpose of this? it seems to me like this should essentialy happen only if the user has deleted the config while running. it would be nice if it showed some error/notification in this case.

Reply:
No, perhaps the config never existed in the first place (which is common on startup)